update_compilers/install_swift_compilers.sh

```diff
@@ -14,11 +14,21 @@ install_nightly() {
 get_swift() {
     local VER=$1
     local DIR=swift-${VER}
+    
+    local BUILD=swift-${VER}-release
+    local TOOLCHAIN=swift-${VER}-RELEASE
+    
+    if [[ "$VER" == "nightly" ]]; then
+        local $(fetch https://swift.org/builds/development/ubuntu1604/latest-build.yml | grep 'download:' | sed 's/:[^:\/\/]/=/g' |
```

Comment:
I've not seen a `local` used like this without a variable assignment. What does this do? From `help local` in bash I think this might not be quite right: perhaps the `local` is unnecessary?

Reply:
the output of the expression is `download=some-toolchain-here` and `local $(download=some-toolchain-here)` evaluates to a local variable named `download` with the value `some-toolchain-here`.

Reply:
I see! It's not at all obvious what variable's being assigned to! I'll merge but I think I'll rephrase a little :) Thanks for clarifying!!

Reply:
maybe we can use [yq](https://yq.readthedocs.io/en/latest/)?